python/test/unit/language/test_matmul.py

```diff
@@ -956,9 +956,10 @@ def block_scale_fp4_matmul(  #
     tl.store(output_ptrs, accumulator, mask=c_mask)
 
 
-@pytest.mark.parametrize("M, N, K", [(1024, 512, 256)])
-@pytest.mark.parametrize("BLOCK_M, BLOCK_N, BLOCK_K", [(128, 128, 128), (256, 128, 128), (128, 256, 128),
-                                                       (128, 256, 256), (128, 128, 64), (128, 64, 128)])
+@pytest.mark.parametrize("M, N, K", [(1024, 512, 512)])
+@pytest.mark.parametrize("BLOCK_M, BLOCK_N, BLOCK_K", [(128, 128, 128), (256, 128, 128),
+                                                       (128, 256, 128), (128, 256, 256), (128, 128, 64), (128, 64, 128),
+                                                       (256, 256, 512)])
```

Comment:
Why adding this block size? This will heavily spills as this test currently uses 4 warps which means the epilogue would need 512 registers.